src/components/ec-file-dropzone/components/ec-upload-cloud-img/ec-upload-cloud-img.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <svg
+    width="50px"
+    height="40px"
+    viewBox="0 0 48 48"
+  >
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M20.3741 5.10425C23.9395 5.10425 27.0697 6.981 28.8442 9.80724C29.8634 9.24871 31.0377 8.93034 32.2878 8.93034C36.1975 8.93034 39.3669 12.0449 39.3669 15.8869C39.3669 16.0047 39.3639 16.1219 39.358 16.2383L39.5396 16.2347C44.2121 16.2347 48 20.05 48 24.7564C48 29.4628 44.2121 33.2782 39.5396 33.2782C39.4817 33.2782 39.424 33.2776 39.3664 33.2764L39.3669 33.2782H7.94245L7.94631 33.2627C3.51315 32.995 0 29.289 0 24.7564C0 20.05 3.78786 16.2347 8.46043 16.2347C9.14224 16.2347 9.80521 16.3159 10.4403 16.4693C10.3868 16.0512 10.3597 15.6244 10.3597 15.1912C10.3597 9.62033 14.8433 5.10425 20.3741 5.10425Z"
+      fill="#DDE2E3"
```

Comment:
`class="tw-fill-gray-6"`

we can also to add `tw-fill-key-4` to the second `path` and remove it from dropzone styles.

Reply:
Updated